go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/confluentinc/cli/v4
 
-go 1.22.7
```

Comment:
@sgagniere I remember we didn't want to bump golang version. Is it still the case or it's safe to do now?

Reply:
Nope, there isn't a golang-fips release yet for 1.23.7.